neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -1596,20 +1606,24 @@ object Neo4jStoreSingleton : GraphStore {
                         it
                     }.linkReversalActionToTransaction(transaction) {
                         paymentProcessor.removeInvoice(it.id)
-                        logger.error(NOTIFY_OPS_MARKER,
-                                """Failed to create or pay invoice for customer ${customer.id}, invoice-id: ${it.id}.
+                        logger.warn(NOTIFY_OPS_MARKER,
+                                """Failed to pay invoice for customer ${customer.id}, invoice-id: ${it.id}.
                                    Verify that the invoice has been deleted or voided in Stripe dashboard.
                                 """.trimIndent())
```

Comment:
Move the starting `"""` to the line above.  Keeping it in the same line prevents timeIndent() from working.

Reference: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-indent.html